ui/page/send/layout.go

```diff
@@ -78,14 +78,33 @@ func (pg *Page) Layout(gtx layout.Context) layout.Dimensions {
 	if pg.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
 		return pg.layoutMobile(gtx)
 	}
+
+	if pg.isModalLayout {
+		modalContent := []layout.Widget{pg.layoutDesktop}
+		return pg.modalLayout.Layout(gtx, modalContent, 450)
+	}
 	return pg.layoutDesktop(gtx)
 }
 
-func (pg *Page) layoutDesktop(gtx layout.Context) layout.Dimensions {
+func (pg *Page) layoutDesktop(gtx layout.Context) D {
 	pageContent := []func(gtx C) D{
 		func(gtx C) D {
 			return pg.pageSections(gtx, values.String(values.StrFrom), false, func(gtx C) D {
-				return pg.sourceAccountSelector.Layout(pg.ParentWindow(), gtx)
+				return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						if pg.isModalLayout {
+							return layout.Inset{
+								Bottom: values.MarginPadding16,
+							}.Layout(gtx, func(gtx C) D {
+								return pg.sourceWalletSelector.Layout(pg.ParentWindow(), gtx)
+							})
+						}
+						return D{}
+					}),
+					layout.Rigid(func(gtx C) D {
+						return pg.sourceAccountSelector.Layout(pg.ParentWindow(), gtx)
+					}),
+				)
```

Comment:
I'm thinking...
```
if !pg.isModalLayout {
    return pg.sourceAccountSelector.Layout(pg.ParentWindow(), gtx)
}
return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
....
```

Reply:
This will mean repeating the wallet selector twice as against setting the a blank dimension.